akka-http-core/src/main/resources/reference.conf

```diff
@@ -463,7 +463,9 @@ akka.http {
     #
     # Set to `infinite` to completely disable entity length checks. (Even then you can still apply one
     # programmatically via `withSizeLimit`.)
-    max-content-length = 8m
+    #
+    # A value of "default" translates to "infinite" for clients and "8m" for servers
```

Comment:
Is "8m" correct? https://github.com/lightbend/config/blob/master/HOCON.md#size-in-bytes-format

Reply:
yes, 8 mebibytes...

Reply:
Hah, you changed to "1024 * 1024" when you made it a constant.

Reply:
Hmm, wouldn't it be more straightforward to change it in the included sections? E.g. we could either comment it out here and set it separately for client and server. Or we keep it as it was here and override it by default in the client section?